python/pyspark/sql/readwriter.py

```diff
@@ -155,21 +155,24 @@ def load(self, path=None, format=None, schema=None, **options):
             return self._df(self._jreader.load())
 
     @since(1.4)
-    def json(self, path, schema=None, primitivesAsString=None, prefersDecimal=None,
+    def json(self, path, schema=None, wholeFile=None, primitivesAsString=None, prefersDecimal=None,
```

Comment:
we need to add this to the end; otherwise it breaks compatibility for positional arguments.